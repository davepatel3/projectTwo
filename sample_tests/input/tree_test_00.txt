float
allocate
-66.864
0
peek_nodes
ref_node
1
allocate
25.2733
1
allocate
20.3392
1
allocate
56.8717
0
allocate
1.69157
0
allocate
94.8021
2
ref_node
6
allocate
-44.2967
6
allocate
-80.3816
0
allocate
83.6205
0
ref_node
9
allocate
-39.5959
1
allocate
47.6719
1
allocate
29.8795
11
allocate
-58.3745
1
allocate
-77.1614
8
peek_nodes
allocate
-98.7144
8
ref_node
15
allocate
-23.8935
13
ref_node
11
peek_nodes
ref_node
10
allocate
-76.7224
2
ref_node
4
ref_node
6
allocate
87.392
5
allocate
-8.76825
4
allocate
-84.1594
10
allocate
84.1165
4
allocate
68.2247
19
allocate
17.1759
8
allocate
-64.0359
7
ref_node
14
ref_node
19
ref_node
24
allocate
27.7907
7
allocate
63.9064
0
ref_node
19
allocate
-10.6756
8
allocate
40.2569
5
allocate
-5.87195
4
allocate
20.5409
14
allocate
17.3895
16
allocate
90.0985
10
allocate
43.9096
7
ref_node
23
allocate
-24.5434
21
allocate
49.205
0
allocate
-89.4641
6
allocate
18.5397
25
allocate
91.4361
32
allocate
-57.1255
14
allocate
-5.18499
28
allocate
34.3762
2
allocate
52.1041
32
ref_node
22
allocate
78.4216
1
allocate
-27.9533
6
ref_node
21
allocate
-6.60648
36
ref_node
4
ref_node
14
allocate
-36.0876
42
ref_node
14
ref_node
6
allocate
-72.5328
32
allocate
96.6522
15
allocate
-89.7573
17
allocate
-87.5645
38
allocate
-48.5056
0
ref_node
51
allocate
41.6038
37
ref_node
38
allocate
39.9401
41
allocate
-27.1809
33
ref_node
28
allocate
-57.6654
0
ref_node
6
allocate
-32.5865
23
ref_node
11
ref_node
24
allocate
30.267
36
allocate
64.6262
56
allocate
-34.4587
45
ref_node
26
allocate
17.8935
13
ref_node
20
ref_node
29
allocate
-48.7045
35
allocate
-83.1559
39
allocate
-40.025
16
allocate
87.9215
20
allocate
23.5048
29
allocate
83.3429
1
ref_node
8
ref_node
57
allocate
-78.6192
54
allocate
-94.5639
46
peek_nodes
ref_node
57
allocate
-79.4925
65
ref_node
42
allocate
7.12105
8
allocate
-23.7676
17
allocate
-88.4771
16
allocate
64.8071
49
peek_nodes
allocate
48.0351
31
peek_nodes
allocate
93.5625
12
allocate
35.116
47
allocate
-87.18
48
peek_nodes
allocate
5.95842
57
ref_node
9
allocate
88.559
63
quit
