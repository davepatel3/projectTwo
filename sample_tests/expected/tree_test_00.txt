allocate
1
peek_nodes
0: 0
1: -66.864
ref_node
-66.864
parent: 0
children: 
allocate
2
allocate
3
allocate
4
allocate
5
allocate
6
ref_node
94.8021
parent: 2
children: 
allocate
7
allocate
8
allocate
9
ref_node
83.6205
parent: 0
children: 
allocate
10
allocate
11
allocate
12
allocate
13
allocate
14
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
allocate
15
ref_node
-98.7144
parent: 8
children: 
allocate
16
ref_node
47.6719
parent: 1
children: 12 
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
15: -98.7144
16: -23.8935
ref_node
-39.5959
parent: 1
children: 
allocate
17
ref_node
56.8717
parent: 0
children: 
ref_node
94.8021
parent: 2
children: 7 
allocate
18
allocate
19
allocate
20
allocate
21
allocate
22
allocate
23
allocate
24
ref_node
-77.1614
parent: 8
children: 
ref_node
-8.76825
parent: 4
children: 22 
ref_node
-64.0359
parent: 7
children: 
allocate
25
allocate
26
ref_node
-8.76825
parent: 4
children: 22 
allocate
27
allocate
28
allocate
29
allocate
30
allocate
31
allocate
32
allocate
33
ref_node
17.1759
parent: 8
children: 
allocate
34
allocate
35
allocate
36
allocate
37
allocate
38
allocate
39
allocate
40
allocate
41
allocate
42
ref_node
68.2247
parent: 19
children: 
allocate
43
allocate
44
ref_node
84.1165
parent: 4
children: 34 
allocate
45
ref_node
56.8717
parent: 0
children: 19 21 29 
ref_node
-77.1614
parent: 8
children: 30 39 
allocate
46
ref_node
-77.1614
parent: 8
children: 30 39 
ref_node
94.8021
parent: 2
children: 7 36 44 
allocate
47
allocate
48
allocate
49
allocate
50
allocate
51
ref_node
-48.5056
parent: 0
children: 
allocate
52
ref_node
91.4361
parent: 32
children: 50 
allocate
53
allocate
54
ref_node
40.2569
parent: 5
children: 40 
allocate
55
ref_node
94.8021
parent: 2
children: 7 36 44 
allocate
56
ref_node
47.6719
parent: 1
children: 12 
ref_node
-64.0359
parent: 7
children: 
allocate
57
allocate
58
allocate
59
ref_node
63.9064
parent: 0
children: 
allocate
60
ref_node
-84.1594
parent: 10
children: 
ref_node
-5.87195
parent: 4
children: 
allocate
61
allocate
62
allocate
63
allocate
64
allocate
65
allocate
66
ref_node
-80.3816
parent: 0
children: 14 15 23 27 
ref_node
30.267
parent: 36
children: 
allocate
67
allocate
68
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
15: -98.7144
16: -23.8935
17: -76.7224
18: 87.392
19: -8.76825
20: -84.1594
21: 84.1165
22: 68.2247
23: 17.1759
24: -64.0359
25: 27.7907
26: 63.9064
27: -10.6756
28: 40.2569
29: -5.87195
30: 20.5409
31: 17.3895
32: 90.0985
33: 43.9096
34: -24.5434
35: 49.205
36: -89.4641
37: 18.5397
38: 91.4361
39: -57.1255
40: -5.18499
41: 34.3762
42: 52.1041
43: 78.4216
44: -27.9533
45: -6.60648
46: -36.0876
47: -72.5328
48: 96.6522
49: -89.7573
50: -87.5645
51: -48.5056
52: 41.6038
53: 39.9401
54: -27.1809
55: -57.6654
56: -32.5865
57: 30.267
58: 64.6262
59: -34.4587
60: 17.8935
61: -48.7045
62: -83.1559
63: -40.025
64: 87.9215
65: 23.5048
66: 83.3429
67: -78.6192
68: -94.5639
ref_node
30.267
parent: 36
children: 
allocate
69
ref_node
52.1041
parent: 32
children: 46 
allocate
70
allocate
71
allocate
72
allocate
73
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
15: -98.7144
16: -23.8935
17: -76.7224
18: 87.392
19: -8.76825
20: -84.1594
21: 84.1165
22: 68.2247
23: 17.1759
24: -64.0359
25: 27.7907
26: 63.9064
27: -10.6756
28: 40.2569
29: -5.87195
30: 20.5409
31: 17.3895
32: 90.0985
33: 43.9096
34: -24.5434
35: 49.205
36: -89.4641
37: 18.5397
38: 91.4361
39: -57.1255
40: -5.18499
41: 34.3762
42: 52.1041
43: 78.4216
44: -27.9533
45: -6.60648
46: -36.0876
47: -72.5328
48: 96.6522
49: -89.7573
50: -87.5645
51: -48.5056
52: 41.6038
53: 39.9401
54: -27.1809
55: -57.6654
56: -32.5865
57: 30.267
58: 64.6262
59: -34.4587
60: 17.8935
61: -48.7045
62: -83.1559
63: -40.025
64: 87.9215
65: 23.5048
66: 83.3429
67: -78.6192
68: -94.5639
69: -79.4925
70: 7.12105
71: -23.7676
72: -88.4771
73: 64.8071
allocate
74
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
15: -98.7144
16: -23.8935
17: -76.7224
18: 87.392
19: -8.76825
20: -84.1594
21: 84.1165
22: 68.2247
23: 17.1759
24: -64.0359
25: 27.7907
26: 63.9064
27: -10.6756
28: 40.2569
29: -5.87195
30: 20.5409
31: 17.3895
32: 90.0985
33: 43.9096
34: -24.5434
35: 49.205
36: -89.4641
37: 18.5397
38: 91.4361
39: -57.1255
40: -5.18499
41: 34.3762
42: 52.1041
43: 78.4216
44: -27.9533
45: -6.60648
46: -36.0876
47: -72.5328
48: 96.6522
49: -89.7573
50: -87.5645
51: -48.5056
52: 41.6038
53: 39.9401
54: -27.1809
55: -57.6654
56: -32.5865
57: 30.267
58: 64.6262
59: -34.4587
60: 17.8935
61: -48.7045
62: -83.1559
63: -40.025
64: 87.9215
65: 23.5048
66: 83.3429
67: -78.6192
68: -94.5639
69: -79.4925
70: 7.12105
71: -23.7676
72: -88.4771
73: 64.8071
74: 48.0351
allocate
75
allocate
76
allocate
77
peek_nodes
0: 0
1: -66.864
2: 25.2733
3: 20.3392
4: 56.8717
5: 1.69157
6: 94.8021
7: -44.2967
8: -80.3816
9: 83.6205
10: -39.5959
11: 47.6719
12: 29.8795
13: -58.3745
14: -77.1614
15: -98.7144
16: -23.8935
17: -76.7224
18: 87.392
19: -8.76825
20: -84.1594
21: 84.1165
22: 68.2247
23: 17.1759
24: -64.0359
25: 27.7907
26: 63.9064
27: -10.6756
28: 40.2569
29: -5.87195
30: 20.5409
31: 17.3895
32: 90.0985
33: 43.9096
34: -24.5434
35: 49.205
36: -89.4641
37: 18.5397
38: 91.4361
39: -57.1255
40: -5.18499
41: 34.3762
42: 52.1041
43: 78.4216
44: -27.9533
45: -6.60648
46: -36.0876
47: -72.5328
48: 96.6522
49: -89.7573
50: -87.5645
51: -48.5056
52: 41.6038
53: 39.9401
54: -27.1809
55: -57.6654
56: -32.5865
57: 30.267
58: 64.6262
59: -34.4587
60: 17.8935
61: -48.7045
62: -83.1559
63: -40.025
64: 87.9215
65: 23.5048
66: 83.3429
67: -78.6192
68: -94.5639
69: -79.4925
70: 7.12105
71: -23.7676
72: -88.4771
73: 64.8071
74: 48.0351
75: 93.5625
76: 35.116
77: -87.18
allocate
78
ref_node
83.6205
parent: 0
children: 
allocate
79
quit
